.travis.yml

```diff
@@ -5,6 +5,18 @@ jdk:
   - oraclejdk8
 sudo: required
 dist: trusty
+cache:
+  directories:
+    - $HOME/mvn-home
+
+install:
+  # install maven 3.3.9 if it is not in cache already
```

Comment:
I'm not sure I see why it matters. The requirement is at least 3.2.5 and 3.3.9 meets that requirement. I suppose we could pin travis to the minimum version but is there any particular reason to do so?